handler.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const uuid = require('uuid');
+const AWS = require('aws-sdk'); 
+AWS.config.setPromisesDependency(require('bluebird'));
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+/* POST /guests */
+module.exports.submitGuest = async (event) => {
+	const requestBody = JSON.parse(event.body);
+  const firstName = requestBody.firstName;
+  const lastName = requestBody.lastName;
+  const responded = requestBody.responded;
```

Comment:
You could use destructuring assignment here like this to do all the assignments at once.
```
const requestBody = JSON.parse(event.body);
const { firstName, lastName, responded } = requestBody;
```
See [Object destructuring on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)